source/create_utplsql_owner.sql

```diff
@@ -31,7 +31,14 @@ create user &ut3_user identified by "&ut3_password" default tablespace &ut3_tabl
 
 grant create session, create sequence, create procedure, create type, create table, create view, create synonym to &ut3_user;
 
-grant execute on dbms_lock to &ut3_user;
+begin
+  $if dbms_db_version.version < 18 $then
+    execute immediate 'grant execute on dbms_lock to &ut3_user';
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a WHEN OTHERS exception handler to ensure that all exceptions are trapped. ([plsql:ExecuteImmediateTrapExceptionsCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3AExecuteImmediateTrapExceptionsCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=820&issues=AWhJ-vvVUMkE2J58C-FU&open=AWhJ-vvVUMkE2J58C-FU)

[ISSUE_KEY]: AWhJ-vvVUMkE2J58C-FU